extension/chrome/elements/compose.htm

```diff
@@ -110,6 +110,11 @@ <h1 id="header_title" data-test="header-title">New Secure Message</h1>
       </tr>
       <tr id="password_or_pubkey_container" data-test="password-or-pubkey-container">
         <td>
+          <div id="expiration_note">
+            Password encrypted messages and attachments <strong>expire in 3 days</strong>.
```

Comment:
The number should be dynamically determined. You'll probably need an API request to figure this out. The logic is as follows:
1) if user does not have authInfo saved, that means they don't have any account with us, therefore must be using the default 3 days
2) if there is `authInfo` present, then it should be used to grab user account info from backend, field `default_message_expire` (see Security screen, where the user can set this property)

By default, default_message_expire may be set to `3` but the user could have changed it to various other possibilities.

One thing, please don't make the API request while composer is loading initially. But it's probably ok to do the request as this infobox is about to render, and delay rendering of the infobox while the request is being sent. I'll leave details up to you.

Reply:
Thanks! Should we also have a link to that setting in the expiation note? Something like "You can change the expiration time __in settings__." where __in settings__ is the link to security settings.

Reply:
You could just say `Recipient will have access to this message <a href=settings>for N days</a>`. Since the user will be looking at it every time he sends a similar message, but is not expected to change it often, I'd make the settings stand out a bit less.

Reply:
Done :+1: